wpimath/src/main/java/edu/wpi/first/math/spline/CubicHermiteSpline.java

```diff
@@ -4,13 +4,29 @@
 
 package edu.wpi.first.math.spline;
 
+import edu.wpi.first.math.spline.proto.CubicHermiteSplineProto;
+import edu.wpi.first.math.spline.struct.CubicHermiteSplineStruct;
+import edu.wpi.first.util.protobuf.ProtobufSerializable;
+import edu.wpi.first.util.struct.StructSerializable;
 import org.ejml.simple.SimpleMatrix;
 
 /** Represents a hermite spline of degree 3. */
-public class CubicHermiteSpline extends Spline {
+public class CubicHermiteSpline extends Spline implements ProtobufSerializable, StructSerializable {
   private static SimpleMatrix hermiteBasis;
   private final SimpleMatrix m_coefficients;
 
+  /** The control vector for the initial point in the x dimension. DO NOT MODIFY THIS ARRAY! */
+  public final double[] xInitialControlVector;
+
+  /** The control vector for the final point in the x dimension. DO NOT MODIFY THIS ARRAY! */
+  public final double[] xFinalControlVector;
+
+  /** The control vector for the initial point in the y dimension. DO NOT MODIFY THIS ARRAY! */
+  public final double[] yInitialControlVector;
+
+  /** The control vector for the final point in the y dimension. DO NOT MODIFY THIS ARRAY! */
+  public final double[] yFinalControlVector;
+
```

Comment:
Could we make these package-private instead of public?  I guess the problem is the struct/protobuf stuff is in a different package, ugh...